scss/bootstrap/overrides/_variables.scss

```diff
@@ -23,10 +23,10 @@ $crm-page-topbar-bg-light:     #393E46;
 $crm-page-topbar-color:        #fff !default;
 $crm-page-topbar-item-padding: 15px !default;
 $crm-accordion-header-color:   #f3f6f7;
-$crm-dasboard-header:          #5D677B;
+$crm-dasboard-header:          #4D4D69;
 $crm-h3-color:                 #525966;
 $crm-wizard-border:            #eff3f5;
-
+$crm-text-muted:               #9494A5;
```

Comment:
why is this necessary? seems yet another additional color

Reply:
This is from palette, we have it for disabled text

Reply:
then definite first as a color `$crm-some-color` and then assign it to the text-muted (where did you get the name?) variable

Reply:
ok, I created name $crm-text-muted my self so, it one can recognize it as muted/disabled text